app/decorators/helpers/spina/application_helper_decorator.rb

```diff
@@ -33,21 +31,19 @@ def crest_class_name(authority)
       end
     end
 
-    def register_description(register, phase)
-      begin
-        response = HTTParty.get("https://#{register}.#{phase}.openregister.org/register.json")
-        response['register-record']['text']
-      rescue
-        "Description not found"
-      end
+    def registers_client
+      RegistersClient::RegistersClientManager.new({ cache_duration: 3600 })
```

Comment:
do we want a shorter cache time?

Reply:
Do you have any suggestions @gidsg ?

Reply:
dunno, how about `600`? we can always tweak this based on feedback.